src/data/constants.js

```diff
@@ -212,3 +214,15 @@ export const ALL_ROUTES = []
   .concat([Routes.DISCUSSIONS.PATH]);
 
 export const MAX_UPLOAD_FILE_SIZE = 1024;
+
+export const topicShape = PropTypes.shape({
+  id: PropTypes.string,
+  usage_key: PropTypes.string,
+  name: PropTypes.string,
+  thread_counts: PropTypes.shape({
+    discussions: PropTypes.number,
+    questions: PropTypes.number,
+  }),
+  enabled_in_context: PropTypes.bool,
+  flags: PropTypes.number,
+});
```

Comment:
`topicShape` is not a constant. Move this to the relevant component and then export from there.